core/modules/path/path.admin.inc

```diff
@@ -642,53 +642,170 @@ function path_patterns_settings_form_submit($form, &$form_state) {
+      $form['update'][$type]['base'] = array(
+        '#type' => 'checkbox',
+        '#title' => t('!label URL aliases', array('!label' => $path_info['label'])),
+        '#attributes' => array(
+          'class' => array('path-base'),
+          'data-path-type' => array($type),
+          'data-path-name' => array('base'),
+        ),
+      );
+      $form['#update_callbacks'][$type] = $path_info;
+      // Check if this entity provides bundle types as pattern items and create
+      // a row for each of these bundle types. For the Node entity the pattern
+      // items would be keyed by be node types; for taxonomy, these would be 
+      // vocabularies.
+      if (isset($path_info['pattern items'])) {
+        foreach ($path_info['pattern items'] as $item => $item_label) {
+          $form['update'][$type][$item] = array(
+            '#type' => 'checkbox',
+            '#title' => t('All !label URL aliases', array('!label' => $item)),
+            '#attributes' => array(
+              'class' => array('path-type', 'path-' . $type),
+              'data-path-type' => array($type),
+              'data-path-name' => array($item),
+            '#parents' => array('update'),
+            ),
+          );
+        }
+      }
     }
   }
 
-  $form['generate']['help'] = array(
-    '#markup' => '<p>' . t('URL aliases will only be created for items that do not already have URL aliases.') . '</p>',
+  // Build the 'Update options' form.
+  $form['operations'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Operations'),
+    '#prefix' => '<div class="container-inline">',
+    '#suffix' => '</div>',
+    '#attributes' => array(
+      'class' => array('path-bulk-operations'),
+    ),
   );
-
-  $form['actions']['#type'] = 'actions';
-  $form['actions']['submit'] = array(
+  $form['operations']['operation'] = array(
+    '#type' => 'select',
+    '#title' => '',
+    '#options' => array(
+      'generate' => t('Generate URL aliases'),
+      'update' => t('Update existing URL aliases'),
+      'delete' => t('Delete URL aliases'),
+    ),
+    '#empty_option' => '- ' . t('Choose an action') . ' -',
+  );
+  $form['operations']['submit'] = array(
     '#type' => 'submit',
-    '#value' => t('Generate URL aliases'),
+    '#value' => t('Execute'),
+    '#submit' => array('path_bulk_update_form_rebuild_submit'),
+    '#validate' => array('path_bulk_update_form_validate'),
   );
 
   return $form;
 }
 
 /**
+ * Returns HTML for the Path bulk update page.
+ *
+ * @param $variables
+ *   An associative array containing:
+ *   - form: A render element representing the form.
+ *
+ * @ingroup themeable
+ */
+function theme_path_bulk_update_form($variables) {
+  backdrop_add_css(backdrop_get_path('module', 'path') . '/css/path.admin.css');
+  backdrop_add_js(backdrop_get_path('module', 'path') . '/js/path.admin.js');
```

Comment:
Looks like `path.admin.js` no longer does anything, we can remove this line and the whole file.